actions/bump-version/action.yaml

```diff
@@ -28,8 +28,8 @@ runs:
     - name: Bump version
       id: release-tag
       run: |
-        echo "oldTag=$(python ./actions/bump-version/scripts/get_version.py)" >> $GITHUB_OUTPUT
+        echo "oldTag=$(python -c "from configparser import ConfigParser; cfg = ConfigParser(); cfg.read('.bumpversion.cfg'); print(cfg['bumpversion']['current_version'])")" >> "$GITHUB_OUTPUT"
```

Comment:
did you confirm that this works? I am a bit concerned about the nested double quotes, that's why we extracted it to an external Python script

Reply:
yes it works: https://github.com/bakdata/github-playground/actions/runs/3788369900
https://github.com/bakdata/github-playground/blob/main/.github/workflows/helm-lint.yaml